src/Tizen.NUI/src/public/DragAndDrop/DragAndDrop.cs

```diff
@@ -224,20 +226,88 @@ public void AddListener(View targetView, DragAndDropEventHandler callback)
             targetEventDictionary.Remove(targetView);
             if (!Interop.DragAndDrop.RemoveListener(SwigCPtr, View.getCPtr(targetView),
                                                     new global::System.Runtime.InteropServices.HandleRef(this, Marshal.GetFunctionPointerForDelegate<Delegate>(cb))))
             {
-                 throw new InvalidOperationException("Fail to RemoveListener");
+                 throw new InvalidOperationException("Fail to RemoveListener for View");
+            }
+        }
+
+        /// <summary>
+        /// Adds listener for drop targets
+        /// </summary>
+        /// <param name="targetWindow">The target Window</param>
+        /// <param name="callback">The callback function to get drag event when the drag source enters, moves, leaves and drops on the drop target</param>
+        public void AddListener(Window targetWindow, DragAndDropWindowEventHandler callback)
+        {
+            InternalDragAndDropEventHandler cb = (dragEvent) =>
+            {
+                DragType type = (DragType)Interop.DragAndDrop.GetAction(dragEvent);
+                DragEvent ev = new DragEvent();
+                global::System.IntPtr cPtr = Interop.DragAndDrop.GetPosition(dragEvent);
+                ev.Position = (cPtr == global::System.IntPtr.Zero) ? null : new Position(cPtr, false);
+
+                if (type == DragType.Enter)
+                {
+                    ev.DragType = type;
+                    callback(targetWindow, ev);
+                }
+                else if (type == DragType.Leave)
+                {
+                    ev.DragType = type;
+                    callback(targetWindow, ev);
+                }
+                else if (type == DragType.Move)
+                {
+                    ev.DragType = type;
+                    callback(targetWindow, ev);
+                }
+                else if (type == DragType.Drop)
+                {
+                    ev.DragType = type;
+                    ev.MimeType = Interop.DragAndDrop.GetMimeType(dragEvent);
+                    ev.Data = Interop.DragAndDrop.GetData(dragEvent);
+                    callback(targetWindow, ev);
+                }
+            };
+
+            targetWindowEventDictionary.Add(targetWindow, cb);
+
+            if (!Interop.DragAndDrop.WindowAddListener(SwigCPtr, Window.getCPtr(targetWindow),
+                                                       new global::System.Runtime.InteropServices.HandleRef(this, Marshal.GetFunctionPointerForDelegate<Delegate>(cb))))
+            {
+                 throw new InvalidOperationException("Fail to AddListener for Window");
+            }
+        }
+
+        /// <summary>
+        /// Removes listener for drop targets
+        /// </summary>
+        /// <param name="targetWindow">The target window</param>
+        /// <param name="callback">The callback function to remove</param>
+        /// <since_tizen> 10 </since_tizen>
```

Comment:
Pleass add hidden mark to avoid ACR requiest.

And.. I dont think it is since API 10.